src/pypgstac/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "pgstacrs"
+version = "0.7.10-dev"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+[lib]
+path = "src/lib.rs"
+name = "pgstacrs"
+crate-type = ["cdylib"]
+
+[dependencies]
+pyo3 = { version = "0.19.1", features = ["abi3-py38", "extension-module"] }
```

Comment:
Note that pyo3 also has a few more features that might be useful to you. In particular `serde`, `anyhow`, `chrono`, and `hashbrown` could be useful. Especially since you're already using `serde` and `thiserror` separately (I think `thiserror` integrates well with anyhow IIRC)